openquake/hazardlib/site.py

```diff
@@ -100,6 +100,13 @@ def __repr__(self):
         return self.__str__()
 
 
+def eq(array1, array2):
+    """
+    Compare two numpy arrays for equality and returns a boolean
+    """
+    return array1.shape == array2.shape and (array1 == array2).all()
```

Comment:
Does the (array1 == array2).all() work if the arrays contain floats. Should they be equal to within a certain precision?